### DISCUSSION TITLE
Replace Characters in Folder Name (While using Folder Behavior: Obsidian Path)

### DISCUSSION THREAD

**Discussion by patppuccin** [ObsidianPublisher/obsidian-github-publisher#104] [Q&A — Plugin]
Hi thanks for this plugin. It has been a real help in building my Docusaurus-based Website, I was wondering if there is any feature to replace characters in the folder name before publishing to GitHub. I know a feature similar to this exists for replacing characters in file names, It would be so great to do the same with folder names.

**Comment by Mara-Li** [comment]
Please, make a issue FR with it.
